internal/services/cosmos/common/indexing_policy.go

```diff
@@ -269,6 +289,7 @@ func ValidateAzureRmCosmosDbIndexingPolicy(indexingPolicy *cosmosdb.IndexingPoli
 
 	if includedPathsDefined {
 		for _, includedPath := range *indexingPolicy.IncludedPaths {
+			// tflog.Debugln("include", *includedPath.Path)
```

Comment:
we should remove this?

Reply:
Thanks for the review @katbyte. This debug line has now been removed in 8e428c4c5c8c8e213926acac885abfc3326dffa3. I've also added unit test in e0bd5f5a343707148bce4477859eac6fcdff256c.